lib/tasks/check_syntax.rake

```diff
@@ -34,7 +34,8 @@ namespace :check do
       end
 
       res = `ruby -c -w #{file}`
-      puts "#{file}: #{res}" if verbose
+      res = res.lines.reject { |s| s == "Syntax OK\n" }.join ""
```

Comment:
No, Ruby does not translate its messages. And I believe that if Matz approved it, a Japanese translation would appear instantly. So this is not a reason to complicate this.